src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -72,6 +74,9 @@ public function getEntriesAction(Request $request)
             throw $this->createNotFoundException();
         }
 
+        $hateoas = HateoasBuilder::create()->build();
+        $json = $hateoas->serialize($entries, 'json');
+
         return $entries;
```

Comment:
Shouldn't you return ` return new Response($json, 200, array('application/json'));` instead ?